apps/judicial-system/backend/src/app/modules/verdict/test/update.spec.ts

```
@@ -0,0 +1,100 @@
+import { uuid } from 'uuidv4'
+
+import { ServiceRequirement } from '@island.is/judicial-system/types'
+
+import { Case } from '../../case/models/case.model'
+import { Defendant } from '../../defendant'
+import { UpdateVerdictDto } from '../dto/updateVerdict.dto'
+import { Verdict } from '../models/verdict.model'
+import { createTestingVerdictModule } from './creatingTestingVerdictModule'
+
+interface Then {
+  result: void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)